### PR TITLE
Removing portal groups and adding dco Team

### PR DESCRIPTION
**Pull request by mrpatrickwest**
Biggest modification is to the person code where I replaced
associatedDCOPortalGroup with the faux property for member of making
sure that the role is MemberRole and the org is a dco:Team. The other
changes are just changing associatedDCOPortalGroup to associatedDCOTeam.

### CODE REVIEW

**Comment by mrpatrickwest** [comment]
This is all working in production. Still need to work on communities before merging the pull request.

File: ingest/mappings/publication.json [hunk 1]
```diff
@@ -167,7 +188,7 @@
         }
       },
       "publicationYear":{
-        "type":"long"
+        "type":"string"
```
Comment (zednis) [inline ingest/mappings/publication.json:191]: Why change publication year to string?  Have we confirmed sorting still works as expected?

File: ingest/ingestHelpers.py [hunk 2]
```diff
@@ -162,9 +162,9 @@ def main(get_objects_query_location, describe_object_query_location,
...
     with open(args.out, "w") as bulk_file:
-        bulk_file.write('\n'.join(ingestSomething.records) + '\n')
+        bulk_file.write('\n'.join(ingestSomething.records) + '\n\n')
 
     # publish the results to elasticsearch if "--publish" was specified on the command line
     if args.publish:
-        bulk_str = '\n'.join(ingestSomething.records) + '\n'
-        ingestSomething.publish(bulk=bulk_str, endpoint=args.es, rebuild=args.rebuild)
\ No newline at end of file
+        bulk_str = '\n'.join(ingestSomething.records) + '\n\n'
```
Comment (zednis) [inline ingest/ingestHelpers.py:169]: Why the double newline?

**Comment by mrpatrickwest** [comment]
Remember we had to add a newline to the end of the bulk file. In this case, though, adding the newline didn't work, but adding two newlines works. Very odd. Adding one newline works everywhere else, just not in datasets.

**Comment by mrpatrickwest** [comment]
tested out the rest of the ingest scripts on dco.tw and they all seem to be working

**Comment by zednis** [comment]
:shipit:

**Pull request merged**